sonar-csharp-plugin/src/main/resources/documentation.md

```diff
@@ -16,12 +16,27 @@ Discover and update the C#-specific [properties](/analysis/analysis-parameters/)
 
 To analyze tool-generated code (e.g. WCF code generated by `SvcUtil.exe`, protobuf code generated by `protoc`, Swagger client code generated by `NSwag`) for a specific C# project, enable the "Analyze generated code" setting inside **Project > Administration > General Settings > C#**. By default, tool-generated code files are skipped from analysis.
 
+The detection of generated code is based on file name, special comments and attributes. The currently recognized values are in [GeneratedCodeRecognizer.cs](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Helpers/GeneratedCodeRecognizer.cs).
+
 ## Scanner compatibility
 
 To analyze C# code, you need to use the [SonarScanner for .NET](/analysis/scan/sonarscanner-for-msbuild/) version 4.x or newer.
 
+## Exclusions
+
+Files excluded from analysis should be set in the project configuration. The files are still going to be analyzed during the compilation and results will be filtered according to the exclusion settings.
```

Comment:
```suggestion
The files which are to be excluded from analysis should be set in the project configuration. Excluded files are still going to be analyzed during the compilation and the results will be filtered according to the exclusion settings.
```

Reply:
(you can also ask @tom-howlett-sonarsource as he's a native English speaker)

Reply:
Files to be exluded....

Reply:
Either read fine. The second starts to feel a bit wordy - "Files **to be** excluded should..." works for me